src/h5web/visualizer/Visualizer.test.tsx

```diff
@@ -91,6 +91,15 @@ describe('Visualizer', () => {
     expect(tabs[0]).toHaveTextContent(Vis.NxImage);
   });
 
+  test('show "NX Image" tab for NXentry group with absolute path to 2D default signal', async () => {
+    renderApp();
+    await selectExplorerNode('nexus_entry/nx_process/absolute_default_path');
+
+    const tabs = await findVisSelectorTabs();
+    expect(tabs).toHaveLength(1);
+    expect(tabs[0]).toHaveTextContent(Vis.NxImage);
+  });
```

Comment:
Testing an absolute default path. This replaces the old `supportsEntity` tests, as mentioned.